src/Chapter1.hs

```diff
@@ -556,7 +562,11 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+  | c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' = True
+  | c == 'A' || c == 'E' || c == 'I' || c == 'O' || c == 'U' = True
```

Comment:
Nice catch on uppercase letters as well 😮 Very thoughtful of you!

Reply:
:)